Lombiq.Tests.UI/Services/OrchardCoreHosting/OrchardApplicationFactory.cs

```diff
@@ -19,6 +19,11 @@
 
 namespace Lombiq.Tests.UI.Services.OrchardCoreHosting;
 
+internal static class OrchardApplicationFactoryCounter
+{
+    public static object CreateHostLock { get; } = new();
```

Comment:
Why does this need to be in a separate class, as opposed to being a static field in `OrchardApplicationFactory`? If it needs to, move this class below that one.

Reply:
See: [S2743](https://rules.sonarsource.com/csharp/RSPEC-2743) A static field in a generic type is not shared among instances of different close constructed types.